modules/gateway/peers.go

```diff
@@ -167,8 +167,8 @@ func (g *Gateway) managedAcceptConnOldPeer(conn net.Conn, remoteVersion string)
 		},
 		sess: muxado.Server(conn),
 	})
-
-	return nil
+	g.addNode(addr)
+	return g.save()
```

Comment:
I am wondering if we still need to support peers with version less than v1.0.0. I know we do not have many left.

Not this PR, just noticing that we still support them and probably don't have to. Just like we did for v0.3.3.x peers, we should probably start pushing <v1 nodes off of the network.